### PR TITLE
PYTHON-3110 [v3.13] Remove use of example.com in CSFLE tests (#848)

### PR DESCRIPTION
**Pull request by blink1073**
(cherry picked from commit 561ee7cf77fcbdefb9e2f46691f2b2ba4c65198b)

### CODE REVIEW

**Comment by blink1073** [comment]
Scheduled encryption runs for Python 2.7 and 3.9

File: test/test_encryption.py [hunk 1]
```diff
@@ -1202,9 +1202,9 @@ def test_06_aws_endpoint_invalid_host(self):
             "region": "us-east-1",
             "key": ("arn:aws:kms:us-east-1:579766882180:key/"
                     "89fcc2c4-08b0-4bd9-9f25-e30687b580d0"),
-            "endpoint": "example.com"
+            "endpoint": "doesnotexist.invalid"
         }
-        with self.assertRaisesRegex(EncryptionError, 'parse error'):
+        with self.assertRaisesRegex(EncryptionError, self.invalid_host_error):
```
Comment (ShaneHarvey) [inline test/test_encryption.py:1207]: invalid_host_error hasn't been defined

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: test/test_encryption.py (hunk 1 above)
Comment (blink1073) [reply test/test_encryption.py:1209]: Fixed

Comment (blink1073) [reply test/test_encryption.py:1209]: And rescheduled encryption tasks

**Comment by blink1073** [comment]
Same issue with Encryption Python 3.8 Amazon Linux 2018 (Enterprise) NoAuth NoSSL, trying with a Windows variant.

**Comment by blink1073** [comment]
Ah, I was misreading the logs.  I fixed the missing `re` import and re-triggered Encryption runs for 2.7 and 3.9

**Review by juliusgeo — APPROVED** [review]
LGTM!

**Pull request merged**